app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt

```diff
@@ -135,9 +142,8 @@ class ToolbarUiController(
 
         layout.turboButton.setImageResource(turboButtonContent.imageId)
 
-        @Suppress("DEPRECATION")
-        toolbarViewModel.legacyState.observe(viewLifecycleOwner, Observer {
-            if (it == null) return@Observer
+        val stateDisposable = toolbarViewModel.state.subscribe {
+            if (it == null) return@subscribe
```

Comment:
it can't be null anymore because it's Rx, right? I wonder if that's indicative of a possible crash here (because it actually can be null) or if this was just to fix the Nullable compiler warning.

Reply:
It's just for the compiler.  Annoying, but better than a suppression.